editor/app/scene/animation-panel.jsx

```diff
@@ -32,21 +33,20 @@ export class AnimationPanel extends React.Component {
     }
 
     return (
-      <React.Fragment>
-        <div className="scene-node-details-header">
+      <Panel>
+        <PanelHeader className="scene-node-details-header">
```

Comment:
@jandrade Im looking to remove CSS definitions from `index.css`. This `scene-node-details-header` is an example where we use a class in basically every panel.  Two main approaches Im thinking about here to move this over to css objects is

1. Create a `SceneNodeDetailsHeader` component which wraps all the styles as a component.
2. Create a `styles.js` file in which we share styles.

I not a huge fan of option 2.  But Im curious about your thoughts and if you have other alternatives!

Reply:
I feel the same about opt. 2 and my vote would be opt. 1 too.

Expanding on what you said about opt. 2, I think that by using a shared `styles.js` file, we end up in the same situation of any `.css` file where we start mixing global styles, so that makes it hard to understand which component uses which style object.

Reply:
awesome - the question answered itself by just moving to css files. I like separate css files for this sort of stuff too.